src/main/java/gregtech/common/metatileentities/electric/multiblockpart/MetaTileEntityFluidHatch.java

```diff
@@ -58,6 +58,15 @@ public NBTTagCompound writeToNBT(NBTTagCompound data) {
         //fluidTank.writeToNBT(data);
         return data;
     }
+    @Override
```

Comment:
same here.
Overriding the fluidInventory with the input tank looks enough to me
```    
protected void initializeInventory() {
        super.initializeInventory();
        this.fluidInventory = importFluids;
}
```